### DISCUSSION TITLE
Working With Secure Email

### DISCUSSION THREAD

**Discussion by jgonzalez210** [O365/python-o365#988] [General]
When attempting to download a email in ".eml" format from a target mailbox what is downloaded is a "message_v2.rpmsg" file which I believe means it was sent as a secure email. 

Was looking through the documentation and did not see any info on handling "rpmsg" files, maybe I should change my approach on how I retrieve the emails?

**Comment by alejcas** [comment]
The eml is downloaded from Ms graph using the mime endpoint. I don't see any info also on using these files. Check ms graph to see if there's something there
